Utils/azure-explorer-common/test/java/com/microsoft/tooling/IntegrationTestBase.java

```diff
@@ -284,28 +305,52 @@ private String applyRegex(String text) {
         }
         return text;
     }
-    
+
     private static Boolean IsMocked() {
-        // TODO: support un mocked mode
+        String keyValue=System.getProperty("isMockedCase");
+        if (keyValue!=null&&keyValue.equalsIgnoreCase("false"))
+            return false;
         return true;
     }
     
+    private synchronized Response redirectErrorHost(Interceptor.Chain chain) throws IOException {
+        Request request=chain.request();
+        String url=chain.request().url().toString();
+        if(IS_MOCKED) {
+            url=url.replace(MOCK_URI, "https://management.notexistazure.com");
```

Comment:
this is the request Url. To simulate the network off situation, send request to non-existed host.